src/always-be-playing.js

```diff
@@ -65,11 +57,15 @@ export default class GapSkipper {
   }
 
   /**
-   * Periodically check for timeupdates to see if a gap has been encountered.
+   * Periodically check current time to see if playback stopped
    *
    * @private
    */
   monitorCurrentTime_() {
+    if (!this.tech_.el_) {
```

Comment:
I have to go back and try to re-create it. I was seeing it happen before, but maybe it was something else. If I can't reproduce, I'll remove the check.